### PR TITLE
CASMPET-6875-release-1.5 update list of known issues in troubleshooting readme

### PR DESCRIPTION
**Pull request by leliasen-hpe**
# Description

The troubleshooting readme does not have the complete list of known issues. Because of this, it is difficult for customers to be aware of all known issues. I am adding the complete list of issues. This fixes [CASMPET-6875](https://jira-pro.it.hpe.com:8443/browse/CASMPET-6875).

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by bo-quan — COMMENTED** [review]
I would remove "Known Issue" from some issue titles to be more consistent and concise.

File: troubleshooting/known_issues/wait_for_unbound_hang.md [hunk 1]
```diff
@@ -31,10 +31,10 @@
             echo $job_id
             job_status=$(echo $job_entry| awk '{ print $2 }')
             echo $job_status
-    	if [[ "$job_status" -eq "0/1" ]];then
+        if [[ "$job_status" -eq "0/1" ]];then
             echo "deleting stale job"
-    		kubectl delete jobs -n services $job_id
+        kubectl delete jobs -n services $job_id
```
Comment (bo-quan) [inline troubleshooting/known_issues/wait_for_unbound_hang.md]: indent issue?

**Review by bo-quan — APPROVED** [review]

File: troubleshooting/known_issues/wait_for_unbound_hang.md (hunk 1 above)
Comment (leliasen-hpe) [reply troubleshooting/known_issues/wait_for_unbound_hang.md]: fixed

**Comment by leliasen-hpe** [comment]
> I would remove "Known Issue" from some issue titles to be more consistent and concise.

I have removed "Known Issue" from the titles.

**Review by rustydb — APPROVED** [review]

**Pull request merged**